src/Modules/Shared/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psm1

```diff
@@ -432,9 +432,9 @@ function Format-Hex
                     # 
                     #            00 01 02 03 04 05 06 07 08 09 0A 0B 0C 0D 0E 0F
                     # 
-                    # 00000000   AA 00 00 00 AB 00 00 00 AC 00 00 00 AD 00 00 00  �...�...�...�...
-                    # 00000010   AE 00 00 00 AF 00 00 00 B0 00 00 00 B1 00 00 00  �...�...�...�...
-                    # 00000020   B2 00 00 00 B3 00 00 00 B4 00 00 00              �...�...�...
+                    # 00000000   AA 00 00 00 AB 00 00 00 AC 00 00 00 AD 00 00 00  ª...«...¬...­...
+                    # 00000010   AE 00 00 00 AF 00 00 00 B0 00 00 00 B1 00 00 00  ®...¯...°...±...
+                    # 00000020   B2 00 00 00 B3 00 00 00 B4 00 00 00              ²...³...´...

```

Comment:
daxian-dbw Sorry. Here is conflict VS Code Editor with special chars. I tried to fix this in other editors but unsuccessfully.
Fortunately this is just a comment. Can remove them?